infra/base-images/base-builder/compile_afl

```diff
@@ -16,8 +16,8 @@
 ################################################################################
 
 # Temporarily disable randomization and enforce a safe and sane setup
-export AFL_SKIP_OSSFUZZ=1
```

Comment:
It does not but however set this made a m mistake which cost afl++ a lot of effectiveness. In the future it is better to talk to me about changes.

Reply:
Ah you talk about the ossfuzz skip, you are right that this enables randomization. When I enabled this it was because nobody told me that this was necessary. It looked like a mistake because the line below is. Classic instrumentation is a very bad idea. In cases where targets do asm stuff a target specific “native” should be used

Reply:
This pr is now 4 months old and I am not at my computer so I have little memory on this … ;)

Reply:
Sure. Let me skip the randomization and try again.